Options/WeakAurasOptions.lua

```diff
@@ -7375,15 +7428,6 @@ function WeakAuras.CreateFrame()
   codereviewCancel:SetWidth(100);
   codereviewCancel:SetText(L["Okay"]);
 
-  function codereview.Close(self)
```

Comment:
Ah ok, makes sense. It should behave the same now when the second one already came after this one